From d57ca61e5486838b823da2e729f2b131697c7c29 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=9F=A9=E6=B6=9B?= <hantao8209@163.com>
Date: Thu, 22 Aug 2024 11:00:32 +0800
Subject: [PATCH] support gpt

 On branch devtool
 Changes to be committed:
	modified:   configs/s32g274ardb2_defconfig
---
 configs/s32g274ardb2_defconfig | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/configs/s32g274ardb2_defconfig b/configs/s32g274ardb2_defconfig
index 8817555955..9c25ae79e4 100644
--- a/configs/s32g274ardb2_defconfig
+++ b/configs/s32g274ardb2_defconfig
@@ -2,8 +2,11 @@ CONFIG_ARM=y
 CONFIG_ARCH_S32G2=y
 # CONFIG_SPI_FLASH_BAR is not set
 CONFIG_SD_BOOT=y
+CONFIG_CMD_GPT=y
+CONFIG_CMD_GPT_RENAME=y
+CONFIG_CMD_UBI=y
+CONFIG_PARTITION_TYPE_GUID=y
+CONFIG_NXP_HSE_SUPPORT=y
 CONFIG_TEE=y
 CONFIG_OPTEE=y
 # CONFIG_OPTEE_TA_AVB is not set
-CONFIG_NXP_HSE_SUPPORT=y
-CONFIG_NXP_HSE_FW_FILE="s32cc_hse_fw.bin"
\ No newline at end of file
-- 
2.42.0

